testing/call_graph_test.c

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
How is this file used?
It is placed in the testing directory but no test is executed for it.

Reply:
I've used it only manually. I am not  familiar enogth with your testing framework.
Can you please suggest how to incorporate it?
Is following good example?
 testing/085/085__tooltip_8cpp.xml
 testing/085_tooltip.cpp

Reply:
I'm not sure that the testing part is really relevant here as you want probably have here a visual comparison of the resulting graph. The tests are there to show differences in, some, basic functionalities.

Reply:
Better to have tests.
I added 
testing/087/087__call__graph_8c_ab50e9ccc8158d9b5cab04dba22244fbd_cgraph.dot testing/087_call_graph.c
But runtests.py doesn't find proper location of dot file. 
@albert-github can you please help to complete it?